client/packages/invoices/src/OutboundShipment/DetailView/SidePanel/TransportSection.tsx

```diff
@@ -6,40 +6,79 @@ import {
   useTranslation,
   useBufferState,
   BufferedTextInput,
+  PanelRow,
+  DateTimePickerInput,
+  DateUtils,
+  Formatter,
 } from '@openmsupply-client/common';
 import { useOutbound } from '../../api';
 
 export const TransportSectionComponent: FC = () => {
   const t = useTranslation();
   const isDisabled = useOutbound.utils.isDisabled();
-  const { transportReference, update } = useOutbound.document.fields([
-    'transportReference',
-  ]);
+  const { transportReference, expectedDeliveryDatetime, update } =
+    useOutbound.document.fields([
+      'transportReference',
+      'expectedDeliveryDatetime',
+    ]);
   const [referenceBuffer, setReferenceBuffer] = useBufferState(
     transportReference ?? ''
   );
 
   return (
     <DetailPanelSection title={t('heading.transport-details')}>
       <Grid container gap={0.5} key="transport-details">
-        <PanelLabel display="flex" alignItems="center">
-          {t('heading.reference')}
-        </PanelLabel>
-        <BufferedTextInput
-          disabled={isDisabled}
-          onChange={e => {
-            setReferenceBuffer(e.target.value);
-            update({ transportReference: e.target.value });
-          }}
-          value={referenceBuffer}
-          slotProps={{
-            input: {
-              style: {
-                backgroundColor: 'white',
+        <PanelRow>
+          <PanelLabel display="flex" alignItems="center" sx={{ flex: 1 }}>
+            {t('label.expected-delivery-date')}
+          </PanelLabel>
+          <DateTimePickerInput
+            disabled={isDisabled}
+            value={DateUtils.getDateOrNull(expectedDeliveryDatetime)}
+            format="P"
+            onChange={expectedDeliveryDatetime => {
+              const formattedDate = expectedDeliveryDatetime
+                ? Formatter.toIsoString(
+                    DateUtils.endOfDayOrNull(expectedDeliveryDatetime)
+                  )?.replace(/Z$/, '')
```

Comment:
Removing the Z at the end of iso string since the nullable input only allows `naivedatetime`

Reply:
Would "getNaiveDate" work here ? We seem to use it for stock and insurance expiry